crash-reporter/init.sls

```diff
@@ -0,0 +1,55 @@
+{% from 'common/map.jinja' import common %}
+{% from tpldir ~ '/map.jinja' import tracker %}
+
+include:
+  - common
+  - python
+
+crash-reporter:
+  virtualenv.managed:
+    - name: /home/servo/crash-reporter/_venv
+    - venv_bin: virtualenv-3.5
+    - python: python3
+    - system_site_packages: False
+    - require:
+      - pkg: python3
+      - pip: virtualenv
+  pip.installed:
+    - pkgs:
+      - git+https://github.com/servo/crash-reporter@{{ tracker.rev }}
+    - bin_env: /home/servo/crash-reporter/_venv
+    - upgrade: True
+    - require:
+      - virtualenv: crash-reporter
+  {% if grains.get('virtual_subtype', '') != 'Docker' %}
+  service.running:
+    - enable: True
+    - name: tracker
+    - require:
+      - pip: crash-reporter
+    - watch:
+      - file: /home/servo/crash-reporter/config.json
+      - file: /lib/systemd/system/tracker.service
+      - pip: crash-reporter
+  {% endif %}
+
+/home/servo/crash-reporter/config.json:
+  file.managed:
+    - source: salt://{{ tpldir }}/files/config.json
+    - template: jinja
+    - user: servo
+    - group: servo
+    - mode: 644
+
+/lib/systemd/system/tracker.service:
+  file.managed:
+    - source: salt://{{ tpldir }}/files/tracker.service
```

Comment:
We'll need a different name for this file or it will overwrite the existing tracker.service. Let's call it `crash-reporter.service` instead.